test/ucs-detect.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/bin/sh
```

There aren't any bash-isms, but it's also not an important suggestion :D

Reply:
NixOS only ensures env and sh so I'm fine either way. :) I can accept this